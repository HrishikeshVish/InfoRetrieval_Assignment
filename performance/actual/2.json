["https://archive.org/details/CNNW_20161218_150000_Fareed_Zakaria_GPS#start/2121/end/2156", "https://archive.org/details/MSNBCW_20190627_070000_Democratic_Candidates_Debate_Post-Debate_Analysis#start/298/end/333", "https://archive.org/details/MSNBCW_20161208_170000_MSNBC_Live_With_Andrea_Mitchell#start/2138/end/2173", "https://archive.org/details/FOXNEWSW_20190812_130000_Americas_Newsroom_With_Bill_Hemmer_and_Sandra_Smith#start/368/end/403", "https://archive.org/details/FOXNEWSW_20190811_200000_Americas_News_HQ#start/2015/end/2050", "https://archive.org/details/MSNBCW_20160927_090000_First_Look#start/2441/end/2476", "https://archive.org/details/MSNBCW_20190516_150000_MSNBC_Live_With_Craig_Melvin#start/1361/end/1396", "https://archive.org/details/MSNBCW_20170112_110000_Morning_Joe#start/2444/end/2479", "https://archive.org/details/MSNBCW_20161208_043000_Hardball_With_Chris_Matthews#start/1264/end/1299", "https://archive.org/details/CNNW_20151231_000000_Erin_Burnett_OutFront#start/1963/end/1998"]