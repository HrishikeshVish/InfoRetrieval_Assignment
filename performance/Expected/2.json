["https://archive.org/details/MSNBCW_20161013_160000_MSNBC_Live#start/1970/end/2005", "https://archive.org/details/FOXNEWSW_20190812_130000_Americas_Newsroom_With_Bill_Hemmer_and_Sandra_Smith#start/368/end/403", "https://archive.org/details/CNNW_20160727_230000_Erin_Burnett_OutFront#start/1668/end/1703", "https://archive.org/details/MSNBCW_20161107_180000_MSNBC_Live_With_Hallie_Jackson#start/1336/end/1371", "https://archive.org/details/CNNW_20160525_040000_CNN_Newsroom_Live#start/4887/end/4922", "https://archive.org/details/CNNW_20161207_160000_At_This_Hour_With_Berman_and_Bolduan#start/408/end/443", "https://archive.org/details/CNNW_20190721_160000_State_of_the_Union_With_Jake_Tapper#start/2711/end/2746", "https://archive.org/details/CNNW_20161218_150000_Fareed_Zakaria_GPS#start/2121/end/2156", "https://archive.org/details/FOXNEWSW_20190811_200000_Americas_News_HQ#start/2015/end/2050", "https://archive.org/details/CNNW_20151231_000000_Erin_Burnett_OutFront#start/1963/end/1998"]